pom.xml

```diff
@@ -16,6 +16,7 @@
         <junit.version>4.12</junit.version>
     </properties>
     <modules>
+        <module>sunbird-user-registry/java</module>
```

Comment:
why we have module name like this  "name/java" Check with @indrajra

Reply:
they have parent pom inside sunbird-user-registry-> java